MinPlatformPkg/Test/Library/PlatformConfigCheckLibNull/DxePlatformConfigCheckLibNull.c

```diff
@@ -0,0 +1,28 @@
+/** @file
+
+  Platform configuration check library NULL implementation for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
+ **/
+
+#include <Uefi.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiLib.h>
+#include <Library/BaseLib.h>
+#include <Library/PrintLib.h>
```

Comment:
It looks like this NULL library instance does not actually need these library dependencies (also remove from INF file).

```suggestion
#include <Uefi.h>
```